my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
```

Comment:
Consider adding a doctoring after all the imports to explain the main purpose of the code. Also, you don't appear to use either glob or os in the code, so you can remove those imports.